File: TUF/Numbers/SumOfGP_Series.ts
function SumOfGP(a: number, r: number, n: number): number {
    let sum: number = 0.0;
    for (let i: number = 0; i < n; i++) {
        sum += a;
        a = a * r;
    }
    return sum;
}

let a: number = 2.0;
let r: number = 1.5;
let n: number = 4;
console.log(SumOfGP(a, r, n));

export { };

File: TUF/Numbers/SumOfGP_Series.java
import java.util.Scanner;

public class SumOfGP_Series {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the value of a: ");
        double a = input.nextDouble();
        System.out.print("Enter the value of r: ");
        double r = input.nextDouble();
        System.out.print("Enter the value of n: ");
        int n = input.nextInt();
        input.close();
        System.out.println(SumOfGP(a, r, n));
    }

    private static double SumOfGP(double a, double r, double n) {
        double sum = 0.0;
        for(int i = 0; i<n; i++) {
            sum += a;
            a = a * r;
        }
        return sum;
    }
}